pkg/rancher-desktop/window/index.ts

```diff
@@ -12,7 +12,7 @@ import paths from '@pkg/utils/paths';
 import { CommandOrControl, Shortcuts } from '@pkg/utils/shortcuts';
 import { openPreferences } from '@pkg/window/preferences';
 
-const console = Logging.background;
+const console = Logging[`window_${ process.type || 'main' }`];
```

Comment:
For… hilarious reasons, `process.type` will never be `main` (it's `browser` for the main process, as opposed to `renderer` for the… renderer process). So maybe make the fallback `unknown` would be better?

Reply:
I'd rather fix these on an as-needed basis for now. `unknown` is a fine alternative to `main` -- `undefined` would be misleading.